instance/models/mixins/openedx_database.py

```diff
@@ -345,6 +345,7 @@ def _get_mysql_settings(self):
             "COMMON_MYSQL_READ_ONLY_PASS": self._get_mysql_pass(self.read_only_user),
             "COMMON_MYSQL_ADMIN_USER": self.admin_user,
             "COMMON_MYSQL_ADMIN_PASS": self._get_mysql_pass(self.admin_user),
+            "edx_django_service_default_db_conn_max_age": 0,
```

Comment:
This doesn't look like it should be under the "Common users" header. Maybe we'll have to make a new header for it and similar vars?

Reply:
Good point, it didn't exactly fit anywhere else.
I created a new section for this variable and other global options.